### PR TITLE
chore(loader): remove classic theme - FE-3092

### PR DESCRIPTION
**Pull request by DipperTheDan**
removal of classic story from loader and includes an MDX example for the Loader component.

fixes FE-3092

### Proposed behaviour

Removal of classic story from the Loader component. 
Updates `StoriesOf` format to Component Story Format(CSF).
Create a new MDX example of the Loader component.

### Current behaviour

Loader component currently has `classic` story examples.

### Checklist

- [x] Commits follow our style guide
~~[ ] Screenshots are included in the PR if useful~~
- [x] All themes are supported if required
~~[ ] Unit tests added or updated if required~~
- [x] Cypress automation tests added or updated if required
- [x] Storybook added or updated if required 
~~[ ] Typescript `d.ts` file added or updated if required~~
- [x] Carbon implementation and Design System documentation are congruent

### Additional context
N/A

### Testing instructions
- Pull down the branch `FE-3092_loader_remove_classic_theme`.
- `npm start` to launch Storybook.
- Navigate to Design System -> Loader.
- For the CSF story example please navigate to Design System -> Loader -> Test

### CODE REVIEW

File: src/components/loader/loader.stories.js [hunk 1]
```diff
@@ -1,57 +1,45 @@
...
+};
 
-storiesOf('Loader', module)
-  .add(...makeStory('default', dlsThemeSelector))
-  .add(...makeStory('classic', classicThemeSelector, true));
+Basic.story = {
+  parameters: {
+    chromatic: {
+      disable: false
+    }
```
Comment (mkrds) [inline src/components/loader/loader.stories.js:43]: Why not remove the whole `Basic.story = {...}` and remove `chromatic: { disabled: true }` from export default?

**Review by KatarzynaQA — DISMISSED** [review]
approve cypress tests

**Comment by mkrds** [comment]
`documentation` folder and `docenInfo.json` should be removed too

File: src/components/loader/loader.stories.js (hunk 1 above)
Comment (DipperTheDan) [reply src/components/loader/loader.stories.js:43]: I did not know that these could be removed. I'll get this addressed. Thank you.

Comment (DipperTheDan) [reply src/components/loader/loader.stories.js]: ✅

File: src/components/loader/loader.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+In general, place a Loader in the centre and middle of the page or container it relates to.
+Like many Design System elements, spinners are made of square elements. They usually take the accent colour of the product they're in.
+
+
+## Quick Start
+Import `Loader` into the project.
+
+```jsx
+
+import { Loader } from "carbon-react/lib/components/loader";
```
[the remark this answers is not in the document]
Comment (DipperTheDan) [reply src/components/loader/loader.stories.mdx:20]: Yes it should be. My bad. I'll get it amended. Thank you.

File: src/components/loader/loader.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+```
+If you require the `Loader` component to be nested inside of the `Button` component, you will also need to import the `Button` component.
+
+```jsx
+
+import { Loader } from "carbon-react/lib/components/loader";
+import { Button } from "carbon-react/lib/components/button";
```
[the remark this answers is not in the document]
Comment (DipperTheDan) [reply src/components/loader/loader.stories.mdx:28]: Yes it should be. My bad. I'll get it amended. Thank you.

File: src/components/loader/loader.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+import Loader from '.';
+import Button from '../button';
+
+<Meta title="Design System/Loader" parameters={{ info: { disable: true }}} />
+
+# Loader
+Use the Loader component to let users know a task or loading data is still in progress. 
+Showing a Loader helps the user to understand that they should wait, rather than reload the page or abandon a process.
+In general, place a Loader in the centre and middle of the page or container it relates to.
+Like many Design System elements, spinners are made of square elements. They usually take the accent colour of the product they're in.
```
Comment (ianforster-sage) [inline src/components/loader/loader.stories.mdx:12]:
```suggestion
Like many Design System elements, Loaders are made of square elements. They usually take the accent colour of the product they're in.
```

**Review by ianforster-sage — CHANGES_REQUESTED** [review]

**Comment by mkrds** [comment]
First commit should also have a `BREAKING CHANGE` body, sth like:

chore(loader): remove classic theme 
BREAKING CHANGE: loader component no longer supports classic theme

**Comment by DipperTheDan** [comment]
> First commit should also have a `BREAKING CHANGE` body, sth like:
> 
> chore(loader): remove classic theme
> BREAKING CHANGE: loader component no longer supports classic theme

It technically isn't a breaking change though. I have not had to remove any styling or component code. The `Classic` story was the only thing that was `classic` about this component.

**Comment by mkrds** [comment]
> > First commit should also have a `BREAKING CHANGE` body, sth like:
> > chore(loader): remove classic theme
> > BREAKING CHANGE: loader component no longer supports classic theme
> 
> It technically isn't a breaking change though. I have not had to remove any styling or component code. The `Classic` story was the only thing that was `classic` about this component.

Yeah, my bad, you're right :)

File: src/components/loader/loader.stories.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (DipperTheDan) [reply src/components/loader/loader.stories.mdx]: ✅

File: src/components/loader/loader.stories.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (DipperTheDan) [reply src/components/loader/loader.stories.mdx]: ✅

File: src/components/loader/loader.stories.mdx (hunk 4 above)
Comment (DipperTheDan) [reply src/components/loader/loader.stories.mdx]: ✅

**Review by mkrds — DISMISSED** [review]

**Review by ianforster-sage — DISMISSED** [review]

**Review by DlgSHi — APPROVED** [review]

**Review by DlgSHi — APPROVED** [review]

**Review by mkrds — APPROVED** [review]

**Review by ianforster-sage — APPROVED** [review]

**Pull request merged**